assets/react/v3/entries/course-builder/components/additional/LiveClass.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { useRef, useState } from 'react';
+import { css } from '@emotion/react';
+import { __ } from '@wordpress/i18n';
+
+import Button from '@Atoms/Button';
+import SVGIcon from '@Atoms/SVGIcon';
+
+import EmptyState from '@Molecules/EmptyState';
+
+import { borderRadius, colorTokens, spacing } from '@Config/styles';
+import { typography } from '@Config/typography';
+import Show from '@Controls/Show';
+import For from '@Controls/For';
+
+import MeetingForm, { MeetingType } from './MeetingForm';
+import MeetingCard from './MeetingCard';
+import { styleUtils } from '@Utils/style-utils';
+import Popover from '@Molecules/Popover';
+import { Portal, usePortalPopover } from '@Hooks/usePortalPopover';
+import { element } from 'prop-types';
+
+export interface Meeting {
+  id: number;
+  type: MeetingType;
+  meeting_title: string;
+  meeting_date: string;
+  meeting_start_time: string;
+  meeting_link: string;
+  meeting_token?: string;
+  meeting_password?: string;
+}
+
+// @TODO: will come from app config api later.
+const isPro = true;
+const hasLiveAddons = true;
+
+const LiveClass = () => {
+  const [showMeetingForm, setShowMeetingForm] = useState<MeetingType | null>(null);
```

Comment:
This variable name is confusing. From the naming it seems like this is a boolean state.